.github/actions/load-img/action.yml

```diff
@@ -2,77 +2,78 @@ name: openverse/load-img
 description: Download and import Docker images from artifacts
 
 inputs:
-  setup_ingestion_server:
-    default: "true"
-    description: Whether to set up ingestion server image
-
-  setup_api:
-    default: "true"
-    description: Whether to set up API image
-
-  setup_catalog:
-    default: "true"
-    description: Whether to set up catalog image
-
-  setup_upstream_db:
-    default: "true"
-    description: Whether to set up upstream DB image
+  setup_images:
+    default: '["upstream_db", "ingestion_server", "catalog", "api", "api_nginx"]'
+    description: Comma-separated list of images (must include trailing comma)
```

Comment:
>(must include trailing comma)

The `default` in the line above doesn't appear to have a trailing comma. Should this read "must not include"?

Reply:
Oh shoot, forgot to update this when I changed from plain-string to JSON in f87f5fe. Updated the description in 31c60f5.